src/index.html

```diff
@@ -9,10 +9,54 @@
     <title>Product cards</title>
     <link
       rel="stylesheet"
-      href="./styles/index.scss"
+      href="./styles/main.scss"
+    />
```

Comment:
The stylesheet link references a `.scss` file. Browsers cannot directly interpret SCSS files; they need to be compiled into CSS first. Ensure that the SCSS file is compiled to CSS, and the link should point to the resulting CSS file.